### PR TITLE
update of translations , removal of presidence couchen

### PR DESCRIPTION
**Pull request by petzlux**

### CODE REVIEW

File: geoportailv3/locale/de/LC_MESSAGES/geoportailv3-client.po [hunk 1]
```diff
@@ -2290,3542 +2270,281 @@ msgstr "Künstliche Wasserkörper 2015"
...
-#: lu_int_wms.1317
-msgid "presidence_stationsservice_station-service_q8_rombach_6"
-msgstr ""
-
-#: lu_int_wms.1318
-msgid "presidence_stationsservice_station-service_q8_rosport"
-msgstr ""
-
-#: lu_int_wms.1319
-msgid "presidence_s
```
Comment (sbrunner) [inline geoportailv3/locale/de/LC_MESSAGES/geoportailv3-client.po]: Relay many delete ... is it normal ?

Comment (jaykayone) [inline geoportailv3/locale/de/LC_MESSAGES/geoportailv3-client.po]: yes, we removed an entire theme from the db, that had 1000+ layers

Comment (sbrunner) [inline geoportailv3/locale/de/LC_MESSAGES/geoportailv3-client.po]: OK :-)

**Comment by sbrunner** [comment]
Looks good for me it can be merge when the travis tests pass :-)

**Pull request merged**